src/impls/mod.rs

```diff
@@ -0,0 +1,10 @@
+#[cfg(feature = "alloc")]
+mod allocs;
+mod arrays;
+mod cores;
```

Comment:
(same here)

```suggestion
mod core;
```

Reply:
also this would name-clash with the reserved `core`.

`cargo check` errors with `error[E0412]: cannot find type 'Iter' in module 'core::slice'`